<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondUI</class>
 <widget class="QMainWindow" name="SecondUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoomKeeper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Ближайшие заезды и выезды</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_booking">
       <property name="text">
        <string>Новая бронь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_in">
       <property name="text">
        <string>Заезд</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_out">
       <property name="text">
        <string>Выезд</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Брони на ближайший месяц</string>
    </property>
    <widget class="Reservations_table" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>541</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Reservations_table</class>
   <extends>QTableWidget</extends>
   <header>reservations/reservations_table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
